### PR TITLE
tsort: add page

### PR DESCRIPTION
**Pull request by MasterOdin**
A command from the list in #2213

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/tsort.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# tsort
+
+> Perform a topological sort.
+
+- Perform a topological sort consistent with a partial sort per line of input broken up into elements separated by blanks:
```
Comment (sbrl) [inline pages/common/tsort.md]: Erm I'm confused! What does this do? That's a lot of words jumbled up in that sentence.

Perhaps we could add some additional clarity in a second line of the command description too - maybe something like this?:

```
> In other words ......
```

Comment (MasterOdin) [reply pages/common/tsort.md]: Yeah, that was my worry when I wrote this.

Essentially, it takes a file where each line is some list of "dependencies":
```
a b c
d
e f
b c d e
```
which can be read as:
- a is before b which is before c
- e is before f
- b is before c which is before d which is before e

and produces a topological sort of it:
```
a
b
c
d
e
f
```

So the gist of that sentence is:
- Produces a topological sort of a file
- Each line of the file gives a partial ordering of elements
- Each element is separated by a space

I'm not sure how wordy of an explanation you guys allow to expand on that singular sentence.

Comment (MasterOdin) [reply pages/common/tsort.md]: The use case would be to see the ordering of dependencies within a directed acyclic graph (a common example is function calls within a program).

Comment (agnivade) [reply pages/common/tsort.md]: We can condense it to - `Perform a topological sort consistent with a partial sort per line of input separated by blanks:`

Comment (sbrl) [reply pages/common/tsort.md]: Ah, ok! Perhaps we could add a 2nd description line saying `A common use is on function calls in a program.`?

**Pull request merged**